From 31ce41a461e8c52f2f4081790d0ff8014f2f3320 Mon Sep 17 00:00:00 2001
From: khalvat-M <mmba313@gmail.com>
Date: Sun, 24 Mar 2024 03:46:23 +0330
Subject: [PATCH 3/3] kernel: Clean headers installed in uapi/asm as well

* Unlike on ARM64, devices building ARM32 installs the signal.h to
  uapi/asm. Include this possible variation to workaround conflicts
  with bionic.

Change-Id: If8e334dfadb544ac3d4256f6738780c797b6cccc
---
 tools/clean_headers.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/clean_headers.sh b/tools/clean_headers.sh
index 627f646..44053a3 100755
--- a/tools/clean_headers.sh
+++ b/tools/clean_headers.sh
@@ -4,4 +4,5 @@ set -eu
 export ANDROID_BUILD_TOP="$PWD"
 
 ./bionic/libc/kernel/tools/clean_header.py -u \
+    "$1/usr/include/asm/signal.h" \
     "$1/usr/include/asm-generic/signal.h"
-- 
2.39.5

